src/peft/tuners/ia3.py

```diff
@@ -155,6 +155,9 @@ def _check_quantization_dependency(self):
                 "To use (IA)^3 with 8-bit quantization, please install the `bitsandbytes` package. "
                 "You can install it with `pip install bitsandbytes`."
             )
+        is_gptq_quantized = getattr(self.model, "is_gptq_quantized", False)
+        if is_gptq_quantized:
+            raise NotImplementedError("GPTQ quantization is not supported for IA3 yet.")
```

Comment:
More of a general comment that `NotImplementedError` is the wrong error to raise (it's intended to be raised by abstract methods), but it's okay to keep it here for consistency.

Reply:
oh thx for the tip